### PR TITLE
chore(docs): Explicit full path when importing cozy-ui component

### PR DESCRIPTION
**Pull request by trollepierre**
- Explicit full path when importing cozy-ui component
- Explicit jsx in code snippets
- Fix typo

Styleguidist: https://github.com/cozy/cozy-ui/pull/2002/files

### CODE REVIEW

File: react/hooks/useBreakpoints/Readme.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ You can use the `useBreakpoints` hook to know whether
 the window has a mobile or desktop width.
 
 A `BreakpointsProvider` must be used above the component using the
-`useBreakpoints` hook to provide it with the breakpoints context.
+`useBreakpoints` hook to provide it with the breakpoints' context.
```
Comment (Ldoppea) [inline react/hooks/useBreakpoints/Readme.md:5]: nit: This is not a `jsx` as described by the commit message 😄

File: react/AppTile/Readme.md [hunk 2]
```diff
@@ -2,9 +2,9 @@ Component used to show an app status, can be used in
 a list of apps for example in the Store.
 
 ```jsx
-import AppTile from '.'
-import mockApps from '../AppSections/_mockApps'
-import { I18n } from '../I18n'
+import AppTile from 'cozy-ui/transpiled/react/AppTile'
+import mockApps from 'cozy-ui/transpiled/react/AppSections/_mockApps'
```
Comment (Ldoppea) [inline react/AppTile/Readme.md:6]: This one is not needed as _mockApps is not part of the public API

File: react/Avatar/Readme.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ import palette from 'cozy-ui/transpiled/react/palette';
 ### With disabled color
 
 ```jsx
-import cozyLogo from '../../docs/cozy-logo_white_128.png'
+import cozyLogo from 'cozy-ui/transpiled/react/../docs/cozy-logo_white_128.png'
```
Comment (Ldoppea) [inline react/Avatar/Readme.md:25]: Same here, everything inside docs folder is not supposed to be in the public API, so original import was better for me

File: react/Avatar/Readme.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ import LinkIcon from "cozy-ui/transpiled/react/Icons/Link";
 ### With overlap
 
 ```jsx
-import cozyLogo from '../../docs/cozy-logo_white_128.png'
+import cozyLogo from 'cozy-ui/transpiled/react/../docs/cozy-logo_white_128.png'
```
Comment (Ldoppea) [inline react/Avatar/Readme.md:42]: Same here

File: react/Avatar/Readme.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ const style={
 The size can also be specifically defined using a number of pixels.
 
 ```jsx
-import cozyLogo from '../../docs/cozy-logo_white_128.png'
+import cozyLogo from 'cozy-ui/transpiled/react/../docs/cozy-logo_white_128.png'
```
Comment (Ldoppea) [inline react/Avatar/Readme.md:66]: Same here

File: react/Hero/Readme.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ import Hero, {
   Icon
 } from 'cozy-ui/transpiled/react/Hero';
 import Button from 'cozy-ui/transpiled/react/Button';
-import { t } from '../../docs/utils';
+import { t } from 'cozy-ui/transpiled/react/../docs/utils';
```
Comment (Ldoppea) [inline react/Hero/Readme.md:15]: Same here

File: react/IntentIframe/Readme.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ Here a simple example :
 
 ```jsx
 import IntentIframe from 'cozy-ui/transpiled/react/IntentIframe'
-import utils from '../../docs/utils'
+import utils from 'cozy-ui/transpiled/react/../docs/utils'
```
Comment (Ldoppea) [inline react/IntentIframe/Readme.md:10]: Same here

File: react/SquareAppIcon/Readme.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ import SquareAppIcon from 'cozy-ui/transpiled/react/SquareAppIcon'
 import CozyIcon from 'cozy-ui/transpiled/react/Icons/Cozy'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import { useCozyTheme } from 'cozy-ui/transpiled/react/CozyTheme'
-import cloudWallpaper from '../../docs/cloud-wallpaper.jpg'
+import cloudWallpaper from 'cozy-ui/transpiled/react/../docs/cloud-wallpaper.jpg'
```
Comment (Ldoppea) [inline react/SquareAppIcon/Readme.md:10]: Same here

File: react/Typography/Readme.md [hunk 9]
```diff
@@ -49,10 +49,10 @@ You can use `color="textSecondary"`:
...
 From old to new :
 
-```
-import { Text, Caption, MainTitle, SubTitle, Title, Bold, ErrorMessage } from '../Text'
-import MuiCozyTheme from '../MuiCozyTheme'
-import Typography from '.';
+```jsx
+import { Text, Caption, MainTitle, SubTitle, Title, Bold, ErrorMessage } from 'cozy-ui/transpiled/react/Text'
+import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
+import Typography from 'cozy-ui/transpiled/react/';
```
Comment (Ldoppea) [inline react/Typography/Readme.md:55]: This should be `import Typography from 'cozy-ui/transpiled/react/Typography'`

File: react/UploadQueue/Readme.md [hunk 10]
```diff
@@ -10,7 +10,7 @@ the upload queue:
 * underneath the top-bar on mobile
 
 ```jsx
-import isTesting from '../helpers/isTesting'
+import isTesting from 'cozy-ui/transpiled/react/helpers/isTesting'
```
Comment (Ldoppea) [inline react/UploadQueue/Readme.md:13]: Same, this helper is for cozy-ui only, it should not be exposed to the public API

**Review by Ldoppea — CHANGES_REQUESTED** [review]

File: react/AppTile/Readme.md (hunk 2 above)
Comment (trollepierre) [reply react/AppTile/Readme.md:6]: nit: next time, use Github suggestion 🙏

File: react/Avatar/Readme.md (hunk 3 above)
Comment (trollepierre) [reply react/Avatar/Readme.md:25]: Capture d’écran 2021-12-24 à 14 23 35

File: react/UploadQueue/Readme.md (hunk 10 above)
Comment (trollepierre) [reply react/UploadQueue/Readme.md:13]: Capture d’écran 2021-12-24 à 14 26 16

File: react/AppTile/Readme.md (hunk 2 above)
Comment (Ldoppea) [reply react/AppTile/Readme.md]: Sadly, the suggestion icon is not accessible when commenting from the "commits" tab :(

File: react/UploadQueue/Readme.md (hunk 10 above)
Comment (Ldoppea) [reply react/UploadQueue/Readme.md]: Doing a project wide search&replace is to agressive here 😅 

This did unwanted changes. I'll point them out on new comments.

File: react/Progress/Readme.md [hunk 11]
```diff
@@ -3,7 +3,7 @@ Progress indicators express an unspecified wait time or display the length of a
 ### Linear and circular progress
 
 ```jsx
-import useProgression from 'cozy-ui/transpiled/react/helpers/useProgression';
+import useProgression from '../helpers/useProgression';
```
Comment (Ldoppea) [inline react/Progress/Readme.md:6]:
```suggestion
import useProgression from 'cozy-ui/transpiled/react/helpers/useProgression';
```

`useProgession` hook may be a public API contrary to `isTesting`

File: react/ProgressionBanner/Readme.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
 A banner with a progress bar at the bottom.
 
 ```jsx
-import useProgression from 'cozy-ui/transpiled/react/helpers/useProgression';
+import useProgression from '../helpers/useProgression';
```
Comment (Ldoppea) [inline react/ProgressionBanner/Readme.md:4]:
```suggestion
import useProgression from 'cozy-ui/transpiled/react/helpers/useProgression';
```

**Review by Ldoppea — CHANGES_REQUESTED** [review]

File: react/Typography/Readme.md [hunk 13]
```diff
@@ -52,7 +52,7 @@ From old to new :
 ```jsx
 import { Text, Caption, MainTitle, SubTitle, Title, Bold, ErrorMessage } from 'cozy-ui/transpiled/react/Text'
 import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
-import Typography from 'cozy-ui/transpiled/react/';
+import Typography from 'cozy-ui/transpiled/react/Typography';
```
Comment (Ldoppea) [inline react/Typography/Readme.md:55]: This one should not be in the `Do not change import for private method` commit

**Review by Ldoppea — APPROVED** [review]
Would be great if you can move the following code to the correct commit.

Everything else is ok for me.

File: react/Typography/Readme.md (hunk 13 above)
Comment (trollepierre) [reply react/Typography/Readme.md:55]: it should be fixed now. I added it in the first commit

**Pull request merged**